src/main/webapp/components/query-selector/query-selector.js

```diff
@@ -82,10 +112,12 @@ const QueryCard = props => {
       >
         <Box style={{ margin: '10px' }}>
           <QueryEditor
-            query={props.query}
-            queryBuilder={AdvancedSearchQueryBuilder}
-            onSearch={query => onSearch({ ...props.query, ...query })}
-            onChange={query => onChange({ ...props.query, ...query })}
+            query={query}
+            queryBuilder={defaultSearchForms[queryBuilder]}
+            queryInteractions={queryInteractions}
+            //TODO only grab non-query-builder properties from exisiting query
```

Comment:
Are there side effects to not having this done?

Reply:
I'm not sure, but it seems safer to grab only the necessary properties rather than everything in the existing query (some which might have been removed by the query builder)